packages/babel-helpers/src/helpers/temporalUndefined.ts

```diff
@@ -2,4 +2,4 @@
 
 // This function isn't mean to be called, but to be used as a reference.
 // We can't use a normal object because it isn't hoisted.
-export default function _temporalUndefined() {}
+export default function _temporalUndefined(): void {}
```

Comment:
```suggestion
export default function _temporalUndefined(this: never): void {}
```

To make sure that this function is not called, since it's not meant to be called.

Reply:
yes!
great idea